www/webapp/static/js/auth/login.js

```diff
@@ -100,7 +100,12 @@ async function loginForm() {
   const data = await response.json();
 
   sessionStorage.setItem("token", data?.token);
-  window.location.href = `/?token=${data?.token}`;
+
+  if(is_redirect_login == "True"){
+    window.location.href = `/?token=${data?.token}`;
+  }else{
+    document.getElementById("btn-submit").classList.add("d-none");
+  }
```

Comment:
I don't understand this why not use the query param

Reply:
i have a doubt to how manage this